ImmuDB4Net/ImmuClient.cs

```diff
@@ -641,6 +891,17 @@ public async Task<List<Entry>> GetAll(List<string> keys)
     // ========== SCAN ==========
     //
 
+    /// <summary>
+    /// Iterates over the key/values in the selected database and retrieves the values for the matching criteria
+    /// </summary>
+    /// <param name="prefix">Optional, prefix for the keys</param>
+    /// <param name="seekKey">Optional, initial key for the first entry in the iteration</param>
+    /// <param name="endKey">Optional, end key for the scanning range</param>
+    /// <param name="inclusiveSeek">Optional, default is true, specifies if initial key's value is included</param>
+    /// <param name="inclusiveEnd">Optional, default is true, specifies if end key's value is included</param>
+    /// <param name="limit">Optional, maximum number of of returned items</param>
```

Comment:
I'd also add a remark that if value `0` is used, the default limit will be used (Which is typically 1000). If returned result is larger than the maximum server-side limit, an error is returned.

Reply:
ok, added this information